ui/js/directives/treeherder/clonejobs.js

```diff
@@ -366,11 +376,32 @@ treeherder.directive('thCloneJobs', [
                     jobCountBtnArray.push(' ');
                 }
             });
+            var output = {
+                "jobBtnArray": jobBtnArray,
+                "jobCountBtnArray": jobCountBtnArray
+            };
+            return output;
+        };
 
+        var addGroupJobsAndCounts = function(jgObj, platformGroup) {
+            var btnArrays = renderGroupJobsAndCounts(jgObj);
+            var jobBtnArray = btnArrays.jobBtnArray;
+            var jobCountBtnArray = btnArrays.jobCountBtnArray;
             platformGroup.find(".group-job-list").html(jobBtnArray);
             platformGroup.find(".group-count-list").html(jobCountBtnArray);
         };
 
+        var getGroupJobsAndCounts = function(jgObj) {
```

Comment:
@wlach , Not so sure what you mean here. Could you give me an example or a link to some sort of guide / documentation? Thanks!